packages/esm-active-visits-app/src/visits-summary/visits-components/tests-summary.component.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { ExtensionSlot } from '@openmrs/esm-framework';
+import { Encounter } from '../visit.resource';
+import styles from '../visit-detail-overview.scss';
+
+const TestsSummary = ({ patientUuid, encounters }: { patientUuid: string; encounters: Array<Encounter> }) => {
+  const filter = React.useMemo(() => {
```

Comment:
Please be consistent. I see:

```
import React from 'react';

/// ...

React.useMemo(...)
```

I also see:

```
import React, { useMemo } from 'react';

/// ...

useMemo(...)
```

Please prefer to use the latter and be consistent.